src/main/java/pl/stormit/tinyurl/dto/UrlAnalyticsLocalizationDto.java

```diff
@@ -0,0 +1,21 @@
+package pl.stormit.tinyurl.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class UrlAnalyticsLocalizationDto {
+
+    private String countryLocalization;
+
+    private String isoCode;
+
+    private String cityLocalization;
+
+    public UrlAnalyticsLocalizationDto() {
+        this.countryLocalization = "";
```

Comment:
I think that NULL value will be enough here.

Reply:
so, just an empty constructor will do the job.

Reply:
Corrected.